soroban-react-dapp/contracts/scripts/deploy.ts

```diff
@@ -23,6 +23,15 @@ export async function deployContracts(addressBook: AddressBook, contracts_to_dep
     console.log(`Contract ID of ${contract_name} is ${contractId}\n\n`)
   }
 
+  // init contract with the admin account
```

Comment:
This script is for any contract we may add to contracts folder, then this initialization will run for every contract we may develop in the future. Not big deal though